src/vcpkg/base/checks.cpp

```diff
@@ -67,6 +71,14 @@ namespace vcpkg
     [[noreturn]] void Checks::exit_with_code(const LineInfo& line_info, const int exit_code)
     {
         Debug::println(locale_invariant_lineinfo(line_info));
+        // Collect exit code and success telemetry
+        get_global_metrics_collector().track_bool(BoolMetric::ExitSuccess, exit_code == EXIT_SUCCESS);
```

Comment:
Is it really easier to have a specific telemetry item for success and a separate item with ; separation rather than just having a point for exit code and a point for location?